BaselineOfPTerm/BaselineOfPTerm.class.st

```diff
@@ -6,15 +6,26 @@ Class {
 
 { #category : #baselines }
 BaselineOfPTerm >> baseline: spec [
+
 	<baseline>
-spec for: #common do: [ 
-	"spec preLoadDoIt: #preload."
-	spec
-		package: 'PTerm-Core' with:  [ spec requires: #('PTerm-Protocol') ];
-		package: 'PTerm-Test';
-		package: 'PTerm-Protocol';
-		package: 'PTerm-Fonts';
-		package: 'PTerm-UI';
-		group: 'default' with: #('PTerm-Core' 'PTerm-Test' 'PTerm-Fonts' 'PTerm-UI')
+	spec for: #common do: [
+		spec
+			package: 'PTerm-Core'
+			with: [ spec requires: #( 'PTerm-Protocol' ) ];
+			package: 'PTerm-Test';
+			package: 'PTerm-Protocol';
+			package: 'PTerm-Fonts';
+			package: 'PTerm-UI';
+			group: 'default' with: #( 'PTerm-Core' 'PTerm-Test' 'PTerm-Fonts' 'PTerm-UI' ) ].
+
+	spec for: #'pharo13.x' do: [
+		spec postLoadDoIt: #postload.
+		spec package: 'PTerm-Pharo13'.
```

Comment:
Same here, for future-proof, I'd avoid naming the package to a specific version of Pharo, Something like `PTerm-SystemInterface` or `PTerm-SystemHook` should be good